R/fa_i.R

```diff
@@ -41,43 +46,13 @@
 #' @export
 fa_i <- function(name,
                  class = NULL,
-                 htmlDependency = NULL,
-                 ...) {
+                 ...,
+                 html_dependency = NULL,
+                 verify_fontawesome = TRUE) {
```

Comment:
What about `verify_fa`? The current name seems really long.

Reply:
Now changed to `verify_fa`.